packages/tocco-ui/src/EditableValue/typeEditors/NumberEdit.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+const NumberEdit = props => {
+  const convertStrintToNumber = stringValue => (stringValue ? Number(stringValue) : null)
```

Comment:
- `convertStringToNumber` (typo)
- can be defined outside of the render function
- Maybe check if string matches number pattern before trying to parse it? (input[type=number] for example still not supported in Opera Mini (global usage 4.36%; according to http://caniuse.com/#feat=input-number))